rules/windows/persistence_user_account_creation.toml

```diff
@@ -11,34 +11,34 @@ domain.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
 name = "User Account Creation"
 risk_score = 21
 rule_id = "1aa9181a-492b-4c01-8b16-fa0735786b2b"
 severity = "low"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.name:(net.exe or net1.exe) and
-  not process.parent.name:net.exe and
-  process.args:(user and (/ad or /add))
+process where event.type in ("start", "process_started") and
+  process.name : ("net.exe", "net1.exe") and
+  not process.parent.name : "net.exe" and
+  (process.args : "user" and process.args : ("/ad", "/add"))
```

Comment:
```suggestion
  (process.args : "user" and process.args : "*/ad*")
```

Reply:
I dont think `/ad` is a valid option. I think the original intent was to look for `*/ad*` to cover add/administrator, etc..

Reply:
```console
$ net user murph_wuz_not_here /ad
The command completed successfully.
```

Reply:
oh nice, and weird